src/api.rs

```diff
@@ -572,6 +574,14 @@ impl<T: Pixel> Context<T> {
     self.frame_count
   }
 
+  pub fn get_skip(&self) -> u64 {
+    self.skip
+  }
+
+  pub fn set_skip(&mut self, skip: u64) {
+     sf.skip = skip;
```

Comment:
self.skip

Reply:
I guess, when I was cleaning up the commit it got messed up